docker/images/n8n/README.md

```diff
@@ -230,6 +230,6 @@ Before you upgrade to the latest version make sure to check here if there are an
 
 ## License
 
-n8n is [fair-code](https://faircode.io) distributed under the [**Sustainable Use License**](https://github.com/n8n-io/n8n/blob/master/packages/cli/LICENSE.md).
+n8n is [fair-code](https://faircode.io) distributed under the [**Sustainable Use License**](https://github.com/n8n-io/n8n/blob/master/LICENSE.md).
```

Comment:
instead of repeating this section, should we link to the `License` section in the root readme?

Reply:
@netroy Something like the below maybe?

```
## License

You can find our [license](https://github.com/n8n-io/n8n/blob/master/README.md#license) information here
```

Reply:
This has been done @netroy